cassandane/testrunner.pl

```diff
@@ -384,10 +383,30 @@ sub usage
 {
     # Build the schedule per commandline
     $plan->schedule(@names);
+
     # Run the schedule
-    open my $fh, '>&', \*STDOUT
-        or die "Cannot save STDOUT as a runner print stream: $!";
-    exit(! $runners{$format}->($plan, $fh));
+    $want_formats{tap} = 1 if not scalar keys %want_formats;
```

Comment:
You don't need that `scalar`, it's implied by `not`.

Reply:
Yeah, I try to be explicit in my intent because it shuts up the "wait, is that a bug???" voice and subsequent rabbit hole dive next time I look at the code after having forgotten writing it -- I'm not in perl often or deeply enough to keep the context rules and exceptions swapped in at all times.  Also theoretically* communicates to the next reader that I considered this and made a deliberate choice, I wasn't merely fumbling pieces together until they worked. ;)

[* It could also communicate the exact opposite, depending what assumptions the reader starts from. Heh /sigh]

Reply:
My advice is "never use `scalar`".  It's usually confusing for an everyday Perl programmer.  "Why was this needed?  What thing here is special?"  Generally you only need it when you do this:

```perl
%hash = (
  key => EXPR,
  xyz => $foo,
);
```

…and EXPR might evaluate to more than one thing.  But that you could address by assigning `EXPR` to a scalar, and using it.  Anyway, this isn't a big deal, but generally "using scalar will make a Perl code reviewer more confused than not". :)